### PR TITLE
Add privacy

### PR DESCRIPTION
**Pull request by sigfrid**
Users can only access their own data. Power-users can access all data.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -94,6 +94,21 @@
   add_index "hours", ["starting_time"], name: "index_hours_on_starting_time", using: :btree
   add_index "hours", ["user_id"], name: "index_hours_on_user_id", using: :btree
 
+  create_table "mileages", force: :cascade do |t|
```
Comment (irismolinari) [inline db/schema.rb]: This table should not be here. We have removed it.

File: spec/features/user_views_entry_audit_trail_spec.rb [hunk 2]
```diff
@@ -1,27 +1,60 @@
...
 
   before(:each) do
     create(:account_with_schema, subdomain: subdomain, owner: user)
-    sign_in_user(user, subdomain: subdomain)
+    add(power_user, subdomain)
   end
 
-  scenario "links to hours audit path" do
-    update_hours_entry
+  context "simple user" do
```
Comment (irismolinari) [inline spec/features/user_views_entry_audit_trail_spec.rb]: We should also forbid to see other users' audit logs, accessible through `hours/HOUR_ID/audits`

Comment (sigfrid) [inline spec/features/user_views_entry_audit_trail_spec.rb]: Done

File: db/schema.rb (hunk 1 above)
Comment (irismolinari) [inline db/schema.rb]: Fixed

**Pull request merged**